mixer/doc/adapter-development-walkthrough.md

```diff
@@ -37,12 +37,13 @@ git clone https://github.com/istio/istio
 
 Install protoc (version 3.5.1 or higher) from [https://github.com/google/protobuf/releases](https://github.com/google/protobuf/releases) and add it to your PATH
 
-Set the MIXER_REPO variable to the path where the mixer repository is on the local machine. Example `export MIXER_REPO=$GOPATH/src/istio.io/istio/mixer`
+Set the MIXER_REPO variable to the path where the mixer repository is on the local machine. Example `export MIXER_REPO=$GOPATH/src/istio.io/istio/mixer`.
+Also, $ISTIO should point to $GOPATH/src/istio.io.
 
-Successfully build the repo.
+Successfully build the mixer repo.
```

Comment:
Let's make this: `Successfully build the mixer server.`.

Reply:
Thanks..Fixed